app/lib/controllers/session.js

```diff
@@ -5,14 +5,14 @@ const Api = require(`${paths.lib.models}/api`)
 
 const Session = function () {}
 
-Session.prototype.authorise = function (username, password, next) {
```

Comment:
@eduardoboucas has this been tested? I seem to recall an issue with changing this parameters name, and that restify wouldn't allow is as an auth middleware routine